cmd/minikube/cmd/start.go

```diff
@@ -790,6 +790,17 @@ func validateUser(drvName string) {
 		exit.Message(reason.DrvNeedsRoot, `The "{{.driver_name}}" driver requires root privileges. Please run minikube using 'sudo -E minikube start --driver={{.driver_name}}'.`, out.V{"driver_name": drvName})
 	}
 
+	// None driver works with root and without root on Linux
+	if runtime.GOOS == "linux" && drvName == driver.None {
+		if !viper.GetBool(interactive) {
+			test := exec.Command("sudo", "-n", "echo", "-n")
+			if err := test.Run(); err != nil {
+				exit.Message(reason.Usage, `sudo requires a password, and --interactive=false`)
```

Comment:
Please use `reason.DrvNeedsRoot` - we have a specific error code that integrators can use to detect this condition.

Reply:
ok